src/components/AttentionBox/__tests__/attentionBox-tests.jest.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import AttentionBox from "../AttentionBox";
+
+describe("AttentionBox tests", () => {
+  const title = "title";
+  const text = "text";
+  let onCloseMock;
+  let attentionBoxComponent;
+  
+  beforeEach(() => {
+    onCloseMock = jest.fn();
+    attentionBoxComponent = render(
+      <AttentionBox onClose={onCloseMock} title={title} text={text}  />
+    );
+  });
+
+  it("should call the click callback when clicked", () => {
```

Comment:
please change the name to: "should call onClose callback when close button clicked"